docs/documentation/siddhi-5.x/siddhi-as-a-kubernetes-microservice-5.x.md

```diff
@@ -52,6 +52,9 @@ siddhi-parser     1         1         1            1           1m
 
 ```
 
+!!! Note "Custom siddhi-runner"
+    If you need to change the `siddhi-runner` image for all the `SiddhiProcess` deployments, you can replace your docker image and image tag with the `SIDDHI_RUNNER_IMAGE` and `SIDDHI_RUNNER_IMAGE_TAG` environment variables in the `siddhi-operator.yaml` file. To create custom siddhi-runner image refer [this documentation](config-guide-5.x.md#adding-to-siddhi-docker-microservice). If your custom docker image from a private Docker registry or repository, then you can specify relevant kubernetes secret by changing `SIDDHI_RUNNER_IMAGE_SECRET` environment variable in the `siddhi-operator.yaml` file. For more details about the private docker images refer [this documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). 
```

Comment:
Shall we rephrase as;
If you need to use a custom-built `siddhi-runner` image for all the `SiddhiProcess` deployments, you have to configure `SIDDHI_RUNNER_IMAGE` and `SIDDHI_RUNNER_IMAGE_TAG` environment variables with the custom-built `image name` and `image tag` respectively in the `siddhi-operator.yaml` file.
Refer the documentation on creating custom siddhi-runner images bundling additional JARs [here](config-guide-5.x.md#adding-to-siddhi-docker-microservice). 
If you are pulling the custom-built image from a private Docker registry/repository, specify the corresponding kubernetes secret as `SIDDHI_RUNNER_IMAGE_SECRET` environment variable in the `siddhi-operator.yaml` file. For more details on using docker images from private registries/repositories refer [this documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).